packages/expo-local-authentication/android/src/main/java/expo/modules/localauthentication/LocalAuthenticationModule.java

```diff
@@ -129,7 +118,16 @@ public void run() {
         mIsAuthenticating = true;
         mPromise = promise;
         mCancellationSignal = new CancellationSignal();
-        mFingerprintManager.authenticate(null, 0, mCancellationSignal, mAuthenticationCallback, null);
+
+        FragmentActivity fragmentActivity = (FragmentActivity) getCurrentActivity();
+        Executor executor = Executors.newSingleThreadExecutor();
+        BiometricPrompt biometricPrompt = new BiometricPrompt(fragmentActivity, executor, mAuthenticationCallback);
+
+        BiometricPrompt.PromptInfo promptInfo = new BiometricPrompt.PromptInfo.Builder()
+                .setDeviceCredentialAllowed(true)
+                .setTitle("Authenticate")
```

Comment:
should `title` (and other fields) be possible to set from JS?

Reply:
Yes, I committed to do a follow up PR on that among other things here: https://github.com/expo/expo/pull/6891#issuecomment-581794377

Still haven't gotten around to doing it though 🙈 might have some time tonight!

Reply:
I am excited to see this get merged into the master. Right now our Android users can only authenticate using fingerprint even if their devices support FaceID. This would be a breaking change.